package.json

```diff
@@ -29,14 +29,15 @@
   },
   "dependencies": {
     "colors": "0.6.x",
+    "mocha": "^2.0.1",
     "optimist": "0.6.x",
     "readcommand": "0.3.x",
     "sprintf-js": "0.0.x",
     "underscore": "1.6.x"
   },
   "devDependencies": {
-    "grunt": "latest",
-    "grunt-mocha-test": "latest",
+    "grunt": "0.4.5",
+    "grunt-mocha-test": "0.12.4",
```

Comment:
I had to fix these dependencies and drop down to v0.10.34 to get the build going again.